packages/forms/src/elements/Range.spec.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import React from 'react';
-import { render, fireEvent } from 'garden-test-utils';
+import { render, fireEvent, prettyDOM } from 'garden-test-utils';
```

Comment:
`prettyDOM` import should be removed

Reply:
Also, the `debug` command returned from `render` uses `prettyDOM` internally if you're looking to use it in the future.

Reply:
🤦‍♀ TY!